front/components/app/blocks/InputBlock.tsx

```diff
@@ -0,0 +1,167 @@
+import { Button, PencilSquareIcon } from "@dust-tt/sparkle";
```

Comment:
Why rename? this breaks the nomemclature. Let's keep the file name and block name ? Will make the PR easier to review as well.

Reply:
No a strong reason enough :)